### DISCUSSION TITLE
v0.0.16 Quart Support and ASGI/async fixes

### DISCUSSION THREAD

**Discussion by cirospaciari** [cirospaciari/socketify.py#101] [Announcements]
### What is changed in socketify:

5% to 10% Faster async in CPython and PyPy

- fix Quart support
- fix ASGI receive HTTP behavior
- fix custom task behavior

### Work in progress for v0.1.0:

- Native optimizations #52 
- Cache tools #60 
- Hot reloading #61

**Full Changelog**: https://github.com/cirospaciari/socketify.py/compare/v0.0.15...v0.0.16

<hr /><em>This discussion was created from the release <a href='https://github.com/cirospaciari/socketify.py/releases/tag/v0.0.16'>v0.0.16 Quart Support and ASGI/async fixes</a>.</em>

**Comment by madkote** [comment]
hi, do you have a roadmap?
I went through issues, and it seems still quite unstable, especially for async.
